src/components/common/AddressBookInput/index.tsx

```diff
@@ -35,6 +35,11 @@ const AddressBookInput = ({ name, canAdd, ...props }: AddressInputProps & { canA
     [addressBookEntries, addressValue],
   )
 
+  const isInAddressBook = useMemo(
```

Comment:
Is this checking the eth address or the displayed name of an entry?

Reply:
Eth address previously assigned to entry.label

Reply:
Thanks! A follow-up question: will it work it I enter a lowercase address but the address book contains a checksummed version of the same address? In some other places we use the sameAddress function for both variants to match.

Reply:
It is working (not sure why tho) but anyways I will use the sameAddress function for consistency :)